### ISSUE TITLE
Lifecicle Event not Triggered on Navigating Back 

### ISSUE THREAD

**Issue by gelinger777** [ionic-team/ionic#18235]
Ionic:

   ionic (Ionic CLI)             : 4.9.0 (/usr/local/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.0.1
   @angular-devkit/build-angular : 0.13.8
   @angular-devkit/schematics    : 7.2.4
   @angular/cli                  : 7.2.4
   @ionic/angular-toolkit        : 1.3.0

Cordova:

   cordova (Cordova CLI) : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms     : browser 5.0.4
   Cordova Plugins       : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 3.1.2, (and 10 other plugins)

System:

   NodeJS : v10.15.3 (/usr/bin/node)
   npm    : 6.9.0
   OS     : Linux 4.15

As you can see I have very fresh Ionic version 4. 

I have a standard Tab App. 

Also I have created a separate Page  . Inisde that page I have a function 

```
goBack(){
  this.navCtrl.navigateForward('/tabs/tab2');

}
```

But when I do it like this , the function 

  ionViewDidEnter()    does not work. It fires only once.

**Comment by liamdebeasi** [comment]
Hi there,

Thanks for the issue. Can you provide a repository with the code required to reproduce this issue? 

Thanks!

**Comment by gelinger777** [comment]
It's a known bug which is just popping over and over since a year. For example #16834 
When you will  take time to understand what is the problem  and fix it ??? You don't need a  repo code. Just generate a standard tabbed app.  put in there the  ionViewDidEnter()  function. Then generate a page out of tab with ionic generate page. From there make a function which  makes 
this.navCtrl.navigateForward('/tabs/tab2');  In the case where you fo to Tab2 from other tabs the  ionViewDidEnter will work, in case you go back from other pages it will not.

**Comment by gelinger777** [comment]
also #8958 the same issue

**Comment by liamdebeasi** [comment]
Hi there,

Thanks for the follow up. I am going to close this since this is a duplicate of #16834 like you mentioned. Please follow that thread for updates. We appreciate your patience as we work to resolve this issue.

Thanks!

**Comment by gelinger777** [comment]
yes, but seems you just abandoned the tickets I mentioned. When you can give a fix for this?

**Comment by liamdebeasi** [comment]
Hi there,

We have not abandoned it, and we are going to prioritize it appropriately. Additionally, we don't usually give estimates on issue resolution.

Thanks!
